technologies/FlexBE/pyrobosim_flexbe_btcpp/config/pyrobosim_flexbe_btcpp.yaml

```
@@ -0,0 +1,9 @@
+bt_action_server:
+  ros__parameters:
+    action_name: "flexbe_bt_server" # Optional (defaults to `bt_action_server`)
+    tick_frequency: 100 # Optional (defaults to 100 Hz)
+    groot2_port: 1667 # Optional (defaults to 1667)
```

```suggestion
    groot2_port: 5555
```

Change to use the same default as Groot2